lib/PaymentIntegrations/CustomIntegration.php

```diff
@@ -31,7 +31,7 @@ public function createFormFields($data = [], $buttonParams = []) {
 
         $data = ['merchant_id' => PayFastPayment::$merchantId, 'merchant_key' => PayFastPayment::$merchantKey] + $data;
 
-        $signature = Auth::generateSignature($data);
+        $signature = Auth::generateSignature($data,PayFastPayment::$passPhrase ?? null);
```

Comment:
Hi,

You're change is good but We already have this in Master branch, and PayFastPayment::$passPhase is by default declared to null.  Basically your pull request was before the change was implemented.
Kind regards